.eslintrc

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
I'll raise the issue in the common repo, but there probably isn't a reason to hide the files in the common repo. That is, a symlink would be:

`ln -s readthedocs-common/eslintrc .eslintrc`

Reply:
https://github.com/rtfd/readthedocs-common/issues/1